.github/workflows/pull-request.yml

```diff
@@ -66,7 +66,7 @@ jobs:
 
   ui_tests:
     name: Visual regression tests
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
```

Comment:
Is this necessary?  If we believe that the regenerated images should be within threshold because of the changed settings below then we should probably try to stay on the cheaper platform if we can.

Reply:
Right you are!